app/views/responsiblepeople/detailed_answers.scala.html

```diff
@@ -115,30 +115,45 @@
                 @checkYourAnswersRow(
                     question = Messages("responsiblepeople.legalName.heading", personName),
                     editUrl = controllers.responsiblepeople.routes.LegalNameController.get(idx, true, flow).toString(),
-                    editLinkTag = "rp-legalname-edit"
+                    editLinkTag = "rp-legalname-yes-no-edit"
+                ) {
+                    @ln.hasPreviousName match {
+                        case Some(false) => {
+                            <p>@Messages("lbl.no")</p>
```

Comment:
We can remove the <p> tags as only needed where two answers are output in the same panel.

Reply:
Removed